synapseclient/api/entity_bundle_services_v2.py

```diff
@@ -0,0 +1,136 @@
+"""This module is responsible for exposing the services defined at:
```

Comment:
Not planning to spend a lot of time on scripts that will exist within this `api` module - The intention is that these will be replaced by a low-level client generated via the OpenAPI spec

Reply:
This is also the start of an initial effort to split functionality out of the client.py. I started with this because there are some upcoming tickets I am aware of that will use these `/bundle2` endpoints. So I used this during my manual testing to make sure that everything was functioning as I expected.